### DISCUSSION TITLE
Generated CSS Cheatsheet

### DISCUSSION THREAD

**Discussion by mccullough-ratehub** [styled-components/styled-components#3791] [Ideas]
Is there a way to generated a CSS cheatsheet from styles defined with `createGlobalStyle`?

**Comment by quantizor** [comment]
This is not a feature styled-components provides. `createGlobalStyle` injects CSS at runtime via `<style>` tags — there is no build-time extraction step that could produce a static CSS file or cheatsheet.

If you want to inspect the CSS that `createGlobalStyle` produces, you can view it in the browser DevTools under the `<style>` tags injected by styled-components (look for `data-styled="active"` in the `<head>`).
